zones/miraheze.wiki

```diff
@@ -27,9 +27,8 @@ $ORIGIN miraheze.wiki.
 ; Servers
 
 ; Services
-status		               CNAME	stats.uptimerobot.com.
 phab		               CNAME	phabricator.miraheze.org.
-status3.miraheze.wiki.         CNAME   miraheze.hund.io.
+status.miraheze.wiki           CNAME    miraheze.hund.io.
```

Comment:
needs to have the . at the end
so like this

status.miraheze.wiki.           CNAME    miraheze.hund.io.

Reply:
or just status per me above :)